lib/mongodb.js

```diff
@@ -262,7 +263,9 @@ MongoDB.prototype.exists = function (model, id, callback) {
   if (self.debug) {
     debug('exists', model, id);
   }
-  id = ObjectID(id);
+  var idType = self.idType(model);
```

Comment:
ditto; `self.idType` is used only once in this block, you do not need to have this line... instead use it like:
```
id = self.idType(id);
```